cmd/images-back/main.go

```diff
@@ -10,29 +10,66 @@ import (
 	"os/signal"
 	"strconv"
 	"time"
+	_ "github.com/lib/pq"
+	"database/sql"
 )
 
 var port int
+var db string
 
 func init() {
-	defPort := 5000
+	defPort := 8080
+	var defDB string
 	if portVar, ok := os.LookupEnv("PORT"); ok {
 		if portValue, err := strconv.Atoi(portVar); err == nil {
 			defPort = portValue
 		}
 	}
+	if dbVar, ok := os.LookupEnv("DATABASE_URL"); ok {
+		defDB = dbVar
+	}
 	flag.IntVar(&port, "port", defPort, "port to listen on")
+	flag.StringVar(&db, "db", defDB, "database to connect to")
 }
 
 func main() {
 	flag.Parse()
 	e := echo.New()
-	e.Static("/static", "static")
 	e.Logger.SetLevel(log.INFO)
+
 	e.GET("/", func(c echo.Context) error {
```

Comment:
I guess this handler will conflict with a handler you've added on [69 line](https://github.com/PhilLar/Images-back/pull/1/files#diff-8051b2d1ba8f5789341d70d783d5dea0R69)